src/redux/Categories/categories.js

```diff
@@ -0,0 +1,15 @@
+const checkStatusAction = '/src/redux/Categories/checkingStatusAction';
```

Comment:
- [ ] Kindly follow the duck pattern for the value of the action type so it will be like :
`bookStore/book/[Action_Name]`.
- [ ] [OPTIONAL] For best practice, Please make all letters of the const capital so it will be like `CHECK_STATUS`.